src/apps/yugabyte/yugabyte_nemesis.py

```diff
@@ -0,0 +1,86 @@
+import random
+from time import sleep
+
+from apps.yugabyte.yugabyte_abstract_app import YugabyteNodeType, AbstractYugabyteApp
+from main.config import Config, ConnectionConfig
+from main.connection import Connection
+
+
+class YugabyteNemesis:
+    def __init__(self,
+                 config: Config,
+                 yugabyte_app: AbstractYugabyteApp):
+        self.config: Config = config
+        self.logger = config.meta.logger
+
+        self.sudo_ssh: Connection = Connection([ConnectionConfig(host, sudo_access=True) for host in config.server_hosts], self.logger)
```

Comment:
```suggestion
        server_hosts = []
        for host in config.server_hosts:
            host['username'] = host.get('sudo_username', host['username'])
            server_hosts.append(ConnectionConfig(host))

        self.sudo_ssh: Connection = Connection(server_hosts, self.logger)
```
And no need for `sudo_access` property